src/components/common/YourCompany/BusinessStartDate.vue

```diff
@@ -189,15 +194,16 @@ export default class BusinessStartDate extends Mixins(CommonMixin, DateMixin) {
   @Getter(useStore) hasBusinessStartDateChanged!: boolean
   @Getter(useStore) isFirmConversionFiling!: boolean
   @Getter(useStore) isFirmCorrectionFiling!: boolean
+  @Getter(useStore) isRoleStaff!: boolean
 
   // Global setter
   @Action(useStore) setCorrectionStartDate!: ActionBindingIF
   @Action(useStore) setValidComponent!: ActionBindingIF
 
-  protected dropdown = false
-  protected onEditMode = false
-  protected isCorrected: boolean = null
-  protected newCorrectedStartDate: string = null // date is "Month Day, Year"
+  dropdown = false
+  onEditMode = false
+  isCorrected: boolean = null
+  private newCorrectedStartDate: string = null // date is "Month Day, Year"
```

Comment:
Why is this private Sev? TypeScript doesn't look at access specifiers right?

Reply:
Same question 👀

Reply:
Well... it sorta does but it's sorta broken, too.

This variable is only used in this script, so `private` makes sense. As such, TS probably won't let code outside this class see this variable (error during development), but it might work at runtime. Another thing that's sorta broken is that variables used in the template should work as private or protected but that gives development errors -- and works anyway.

For this particular variable, its usage is clear (and it's not used in the template), so private makes sense.

Reply:
OK, makes a lot of sense. Keeping this in mind moving forward. Thanks!

Reply:
This is one of those gray areas where experience can help you make a decision 😁